### PR TITLE
k3s: fix 1.1.11 check for all the profiles

### PR DESCRIPTION
**Pull request by vardhaman22**
issues: https://github.com/rancher/cis-operator/issues/350 https://github.com/rancher/rancher/issues/46663
fixed 1.1.11 check for k3s
- updated remediation
- updated audit command (removed echo statement)
- set scored: false and added manual in the check text

### CODE REVIEW

File: package/cfg/k3s-cis-1.23-hardened/master.yaml [hunk 1]
```diff
@@ -154,25 +154,22 @@ groups:
...
+        text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Manual)"
+        audit: stat -c permissions=%a $etcddatadir
         tests:
           test_items:
             - flag: "permissions"
               compare:
                 op: bitmask
                 value: "700"
         remediation: |
+          Not Applicable for the non-etcd cluster. If running master only with no etcd role, this check is Not applicable.
```
Comment (pjbgf) [inline package/cfg/k3s-cis-1.23-hardened/master.yaml:166]:
```suggestion
          Not applicable for the non-etcd cluster. If running master only with no etcd role, this check is not applicable.
```

File: package/cfg/k3s-cis-1.23-permissive/master.yaml [hunk 2]
```diff
@@ -154,25 +154,22 @@ groups:
...
+        text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Manual)"
+        audit: stat -c permissions=%a $etcddatadir
         tests:
           test_items:
             - flag: "permissions"
               compare:
                 op: bitmask
                 value: "700"
         remediation: |
+          Not Applicable for the non-etcd cluster. If running master only with no etcd role, this check is Not applicable.
```
Comment (pjbgf) [inline package/cfg/k3s-cis-1.23-permissive/master.yaml:166]:
```suggestion
          Not applicable for the non-etcd cluster. If running master only with no etcd role, this check is not applicable.
```

File: package/cfg/k3s-cis-1.24-hardened/master.yaml [hunk 3]
```diff
@@ -148,25 +148,22 @@ groups:
...
+        text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Manual)"
+        audit: stat -c permissions=%a $etcddatadir
         tests:
           test_items:
             - flag: "permissions"
               compare:
                 op: bitmask
                 value: "700"
         remediation: |
+          Not Applicable for the non-etcd cluster. If running master only with no etcd role, this check is Not applicable.
```
Comment (pjbgf) [inline package/cfg/k3s-cis-1.24-hardened/master.yaml:160]:
```suggestion
          Not applicable for the non-etcd cluster. If running master only with no etcd role, this check is not applicable.
```

File: package/cfg/k3s-cis-1.24-permissive/master.yaml [hunk 4]
```diff
@@ -149,25 +149,22 @@ groups:
...
+        text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Manual)"
+        audit: stat -c permissions=%a $etcddatadir
         tests:
           test_items:
             - flag: "permissions"
               compare:
                 op: bitmask
                 value: "700"
         remediation: |
+          Not Applicable for the non-etcd cluster. If running master only with no etcd role, this check is Not applicable.
```
Comment (pjbgf) [inline package/cfg/k3s-cis-1.24-permissive/master.yaml:161]:
```suggestion
          Not applicable for the non-etcd cluster. If running master only with no etcd role, this check is not applicable.
```

File: package/cfg/k3s-cis-1.7-hardened/master.yaml [hunk 5]
```diff
@@ -149,25 +149,22 @@ groups:
...
+        text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Manual)"
+        audit: stat -c permissions=%a $etcddatadir
         tests:
           test_items:
             - flag: "permissions"
               compare:
                 op: bitmask
                 value: "700"
         remediation: |
+          Not Applicable for the non-etcd cluster. If running master only with no etcd role, this check is Not applicable.
```
Comment (pjbgf) [inline package/cfg/k3s-cis-1.7-hardened/master.yaml:161]:
```suggestion
          Not applicable for the non-etcd cluster. If running master only with no etcd role, this check is not applicable.
```

File: package/cfg/k3s-cis-1.7-permissive/master.yaml [hunk 6]
```diff
@@ -149,25 +149,22 @@ groups:
...
+        text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Manual)"
+        audit: stat -c permissions=%a $etcddatadir
         tests:
           test_items:
             - flag: "permissions"
               compare:
                 op: bitmask
                 value: "700"
         remediation: |
+          Not Applicable for the non-etcd cluster. If running master only with no etcd role, this check is Not applicable.
```
Comment (pjbgf) [inline package/cfg/k3s-cis-1.7-permissive/master.yaml:161]:
```suggestion
          Not applicable for the non-etcd cluster. If running master only with no etcd role, this check is not applicable.
```

File: package/cfg/k3s-cis-1.8-hardened/master.yaml [hunk 7]
```diff
@@ -149,25 +149,22 @@ groups:
...
+        text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Manual)"
+        audit: stat -c permissions=%a $etcddatadir
         tests:
           test_items:
             - flag: "permissions"
               compare:
                 op: bitmask
                 value: "700"
         remediation: |
+          Not Applicable for the non-etcd cluster. If running master only with no etcd role, this check is Not applicable.
```
Comment (pjbgf) [inline package/cfg/k3s-cis-1.8-hardened/master.yaml:161]:
```suggestion
          Not applicable for the non-etcd cluster. If running master only with no etcd role, this check is not applicable.
```

File: package/cfg/k3s-cis-1.23-hardened/master.yaml (hunk 1 above)
Comment (vardhaman22) [reply package/cfg/k3s-cis-1.23-hardened/master.yaml]: done

File: package/cfg/k3s-cis-1.23-permissive/master.yaml (hunk 2 above)
Comment (vardhaman22) [reply package/cfg/k3s-cis-1.23-permissive/master.yaml]: done

File: package/cfg/k3s-cis-1.24-hardened/master.yaml (hunk 3 above)
Comment (vardhaman22) [reply package/cfg/k3s-cis-1.24-hardened/master.yaml]: done

File: package/cfg/k3s-cis-1.24-permissive/master.yaml (hunk 4 above)
Comment (vardhaman22) [reply package/cfg/k3s-cis-1.24-permissive/master.yaml]: done

File: package/cfg/k3s-cis-1.7-hardened/master.yaml (hunk 5 above)
Comment (vardhaman22) [reply package/cfg/k3s-cis-1.7-hardened/master.yaml]: done

File: package/cfg/k3s-cis-1.7-permissive/master.yaml (hunk 6 above)
Comment (vardhaman22) [reply package/cfg/k3s-cis-1.7-permissive/master.yaml]: done

File: package/cfg/k3s-cis-1.8-hardened/master.yaml (hunk 7 above)
Comment (vardhaman22) [reply package/cfg/k3s-cis-1.8-hardened/master.yaml]: done

**Review by pjbgf — APPROVED** [review]

**Review by andypitcher — APPROVED** [review]

**Pull request merged**